crates/cheatnet/src/runtime_extensions/forge_runtime_extension/cheatcodes/declare.rs

```diff
@@ -68,8 +69,7 @@ pub fn declare(
     }
 }
 
-pub fn get_class_hash(sierra_contract: &str) -> Result<ClassHash> {
-    let sierra_class: SierraClass = serde_json::from_str(sierra_contract)?;
+pub fn get_class_hash(sierra_class: &SierraClass) -> Result<ClassHash> {
```

Comment:
Changed the signature to parse sierra less times. In fact we still parse it too much, it should be parsed when loading contract artifacts - #1728